linera-base/src/data_types.rs

```diff
@@ -1000,6 +1002,11 @@ impl BlobContent {
         BlobContent::new(BlobType::ApplicationDescription, bytes)
     }
 
+    /// Creates a new committee [`BlobContent`] from the provided serialized committee.
+    pub fn new_committee(committee: impl Into<Box<[u8]>>) -> Self {
```

Comment:
This API should be more type-safe. Right now it accepts any bytes blindly a s Committee which can led to problems. Why not pass the actual committee instead and serialise it here?

Reply:
I created https://github.com/linera-io/linera-protocol/issues/3520 for this.